### ISSUE TITLE
Seller doesn't see my Bid

### ISSUE THREAD

**Issue by MatrixManAtYrService** [particl/particl-desktop#1698]
# Seller doesn't see my Bid

When I view the "Buy Orders" page my order (Hash: d45dad) appears in the bidding status.  It has been that way for a few weeks.  The seller put his e-mail address on the product description, so I e-mailed him to remind him that this was his turn.  He got back to me and says that he doesn't see my order.

**To Reproduce**

As Buyer:
 - find a listing
 - bid on it

As Seller:
 - [unsure what it looks like to seller, I can ask him to join this thread if needed]

**Expected behavior**

Seller has the option to accept my bid

**Screenshots**

https://imgur.com/a/3kerBrh

 - OS: Arch Linux (Linux 5.4.15-arch1-1 x86_64)
 - Particl Desktop: 2.3.2 (beta)
 - Particl Core: 0.18.1.16

**Additional context**

Not being on both sides of the transaction, it's hard to make this a better bug report.  What can I do to make this easier to diagnose?

**Comment by kangoala** [comment]
Hi MatrixMan,
I once had the same problem with my first marketplace installation.
There I recovered my wallet by seed. Did you do anything similar before? Does other orders/smsg-messages arrive at the sellers place?

**Comment by MatrixManAtYrService** [comment]
@kangoala Yes actually.  I locked myself out of my wallet by forgetting the password.  I worked around this by deleting `~/.particl/wallet.dat` and recovering it from seed.  I had forgotten that this might be a relevant detail--sorry for not sharing it initially.

I tried to order one other thing, but cancelled it after only a few days.  I assumed that the seller was just unresponsive.  Now I'm thinking that they never saw it.  I haven't tried to order anything else.

I'm not sure what `smsg-messages` are, but I'm happy to test this out if somebody can give me some guidance.

**Comment by kangoala** [comment]
If other sellers/buyers may seem unresponsive, this could be related to the fact that they don't receive the notifications (messages) of their counterpart (that was meant with smsg-messages). There's a manual way to resend those messages, but I guess this won't help in your case, since the messages from your seller probably gets somehow "misdirected" and don't arrive in your freshly recovered marketplace installation.
That's clearly not how it should has to be, but the marketplace is still in a very early state and I guess the devs will address this issue as soon as possible. Right now, devs are still busy with the upcoming marketplace management implementation, thus I would recommend you to withdraw your coins from your latest marketplace installation, and restart with a freshly generated seed. I'm not sure, but probably to keep the default wallet should be fine. 
I had the chance to make another installation on a second PC, and there everything works the way it has to be. If you feel uncomfortable on how to proceed, just leave here another comment. I am sure one of the devs will further assist you then.
Good luck and hopefully a better shopping experience afterwards :-)

**Comment by zaSmilingIdiot** [comment]
Checking in if this is still an applicable issue with the release of v3.0? And more specifically, with the release of v3.1.x wherein the ability to resend notifications automatically have been added?
